src/pages/ShowFeelingsPage/ShowFeelingsPage.tsx

```diff
@@ -48,8 +56,9 @@ export const ShowFeelingsPage = () => {
               </h3>
               <ShowFeelingTemplate
                 key={feelingsList[date]}
-                feelingCategory="Test"
                 feelingsListObject={feelingsList[date]}
+                setFeelingsListObject={handleFeelingsListChange}
```

Comment:
Do not handle/pass the parent state like this if you want to change it from child.
It should be                 
setFeelingsListObject={{ feelingsList, setFeelingsList }}

Reply:
Thanks for the suggestion! Actually, this is the way to do it when you are using something like recoil for state management of these lists i.e., this is the performant way mentioned in react docs. But for props, it doesn't really work apparently.

Reply:
well, let's see if we encounter any such scenario in the future. here it works at least. ;)